src/transport/SecureSession.h

```diff
@@ -169,7 +170,20 @@ class SecureSession : public Session
     }
 
     System::Clock::Timestamp GetLastActivityTime() const { return mLastActivityTime; }
+    System::Clock::Timestamp GetLastPeerActivityTime() const { return mLastPeerActivityTime; }
     void MarkActive() { mLastActivityTime = System::SystemClock().GetMonotonicTimestamp(); }
+    void MarkActiveRx()
+    {
+        mLastPeerActivityTime = System::SystemClock().GetMonotonicTimestamp();
+        MarkActive();
+    }
+
+    bool IsPeerActive() { return ((System::SystemClock().GetMonotonicTimestamp() - GetLastPeerActivityTime()) < kMinActiveTime); }
```

Comment:
This check matches the spec, but I am still not clear on why the spec thinks this makes sense.  Min active time is not defined to start after "last activity" in the spec.... or has that been changed?

Reply:
Yes, updated to last network activity seen from peer.